features/filmStrip/components/FilmStrip.js

```diff
@@ -38,12 +38,13 @@ class FilmStrip extends Component {
                                 <VideoThumbnail
                                     key={p.id}
                                     participant={p}
-                                    videoTrack={videoTrack}
-                                    audioTrack={audioTrack}
                                     audioMuted={!audioTrack ||
                                         (audioTrack && audioTrack.isMuted())}
+                                    audioTrack={audioTrack}
+                                    avatar={p.avatar}
```

Comment:
This might be connected to previous problem with Component not updating, but "VideoThumbnail" already has a participant from which we can get the avatar, so I think we don't have to pass "avatar" from outside